integrations/download_test.go

```diff
@@ -70,24 +71,26 @@ func TestDownloadRawTextFileWithoutMimeTypeMapping(t *testing.T) {
 
 	session := loginUser(t, "user2")
 
-	req := NewRequest(t, "GET", "/user2/repo2/raw/branch/master/test.xml")
+	req := NewRequest(t, "GET", "/user2/repo2/raw/branch/master/bin.foo")
```

Comment:
Could you just add a second test case instead of modifying the existing one?

Reply:
Good to know.

Reply:
Yeah, it's generally good to only add tests, never remove them. That way you can be sure to not regress those existing tests.